Odata-docs/client/using-blazor-wasm-with-odata-client.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Blazor WebAssembly with OData Client"
+description: "How to use Blazor with OData Client"
+
+author: ElizabethOkerio
+ms.author: elokerio
+ms.date: 1/18/2021
+ms.topic: article
+ 
+---
+# How To Use Blazor WebAssembly with OData Client 
+**Applies To**: [!INCLUDE[appliesto-odataclient](../includes/appliesto-odataclient-v7.md)]
+
+## Getting Started
+Start by creating a Blazor WebAssembly project. Follow the following steps to create a Blazor WebAssembly project. 
+
+* Open Visual Studio and click on "Create a new project".
+* On the window that appears, select the "Blazor App" tempate and click "Next". 
+* On the "Configure your project" page, configure your project by providing the project name and the location path then click "Create".
+* Finally on the "Create Project" page, specify the target framework that you want to use and select "Blazor WebAssembly App" then click "Create". 
+
+A Blazor Web Assembly project will be created like below. 
+
+![SAMPLE BLAZOR WEB ASSEMBLY PROJECT](../assets/2021-02-02-blazor-wasm-app-sample.png)
+
+There are two ways in which you can use OData Client with Blazor WebAssembly: 
+
+* You can use the Microsoft OData Connected Service
+* You can use plain-old C# objects (POCOs)
+
+## Using Microsoft OData Connected Service
+
+Add OData Connected Service to the project you just created above. 
+
+To do so, ensure that you have the Microsoft OData Connected Service extension installed. Refer to this link [Microsoft OData Connected Service](/odata/connectedservice/getting-started/) to find out how get started with Microsoft OData Connected Service.
+Right click on your project and add a Connected Service. Select the Microsoft OData Connected Service that you installed and use the Wizard to generate proxy classes for the OData endpoint that you want to use.
+
+Once all this is done, open any of the Blazor Razor pages on which you want to display data from an OData endpoint and include the code below in the `OnInitializedAsync` method.  
+
+The use of the below syntax is possible with `Microsoft.OData.Client v7.8.1` and above. 
+
+```c#
+Uri serviceRoot = new Uri("https://localhost:44353/odata");
+var dataServiceContect = new Default.Container(serviceRoot);
+dataServiceContect.HttpRequestTransportMode = HttpRequestTransportMode.HttpClient;
+var result = await dataServiceContect.Books.ExecuteAsync();
+```
+You can then display the data however you want on your Razor pages.  
```

Comment:
In the [section below](https://github.com/MicrosoftDocs/OData-docs/pull/169/files#diff-0295219f7f0d0c74c67044e3d2144690c7261a6bee0611c1b14be57eff79a424R85) you iterate through `result` and render `Author` and `ISBN`. Maybe you could do the same here...